### PR TITLE
nixos/atlassian-jira: Drop postgres requirement and fix capitalization

### PR DESCRIPTION
**Pull request by dasJ**
###### Motivation for this change

See each commit message for more details.
cc @fpletz @globin 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by dasJ** [comment]
@GrahamcOfBorg eval
↑ because another package was broken

File: nixos/modules/services/web-apps/atlassian/jira.nix [hunk 1]
```diff
@@ -170,7 +170,6 @@ in
       description = "Atlassian Jira";
 
       wantedBy = [ "multi-user.target" ];
-      requires = [ "postgresql.service" ];
```
Comment (globin) [inline nixos/modules/services/web-apps/atlassian/jira.nix]: requires = mkIf services.postgresql.enable [ "postgresql.service" ];

**Review by globin — CHANGES_REQUESTED** [review]

File: nixos/modules/services/web-apps/atlassian/jira.nix (hunk 1 above)
Comment (WilliButz) [reply nixos/modules/services/web-apps/atlassian/jira.nix:173]: Instead of `cfg` this should be `config`.

**Review by WilliButz — CHANGES_REQUESTED** [review]
One minor change request, otherwise this looks good to merge :+1:

**Pull request closed without merging**